src/OpticalFlowEstimator.cpp

```diff
@@ -255,6 +261,58 @@ bool OpticalFlowEstimator::canEstimateFlow() const
         return false;
     }
 
+    double yaw, pitch, roll;
```

Comment:
All from here down is duplicate code (written by yours truly for POC) needs to be cleaned.